src/main/java/org/aksw/gerbil/database/ExperimentTaskResultRowMapper.java

```diff
@@ -54,15 +54,15 @@
     public ExperimentTaskResult mapRow(ResultSet resultSet, int rowId) throws SQLException {
         int idInDatabase = -1;
         try {
-            idInDatabase = resultSet.getInt(14);
+            idInDatabase = resultSet.getInt(15);
         } catch (Exception e) {
             // nothing to do
         }
         return new ExperimentTaskResult(resultSet.getString(1), resultSet.getString(2),
                 ExperimentType.valueOf(resultSet.getString(3)), Matching.valueOf(resultSet.getString(4)),
                 new double[] { resultSet.getDouble(5), resultSet.getDouble(6), resultSet.getDouble(7),
                         resultSet.getDouble(8), resultSet.getDouble(9), resultSet.getDouble(10) },
-                resultSet.getInt(11), resultSet.getInt(12), resultSet.getTimestamp(13).getTime(), idInDatabase);
+                resultSet.getInt(11), resultSet.getInt(12), resultSet.getTimestamp(13).getTime(), idInDatabase, resultSet.getString(14));
```

Comment:
@CatherineRomiyo did you tested this? I am not sure whether it is working.

Reply:
@MichaelRoeder I have changed the column index for idInDatabase as 15. If I change the column index for version to 15 it is showing column not found. So I changed it here (https://github.com/dice-group/gerbil/pull/271/commits/6118531e661a607b8025f4f8099dc72639aaa74d)
//Before  
idInDatabase = resultSet.getInt(14);
//After
idInDatabase = resultSet.getInt(15);
After this change code is working fine. Kindly have a look at the commit mentioned above and let me know if its wrong.